goupnp.go

```diff
@@ -123,6 +123,15 @@ func DeviceByURL(loc *url.URL) (*RootDevice, error) {
 	return root, nil
 }
 
+type CharsetReader func(charset string, input io.Reader) (io.Reader, error)
+
+var charsetReaderDefault CharsetReader
+
+// Init sets the default charsetReader for UPnP clients, allowing for non-utf8 encodings.
+func Init(charsetReader CharsetReader) {
```

Comment:
You're probably right in your comment, maybe make the variable public instead. Just make a note not to change it while goupnp libraries are in use.